project/Dependencies.scala

```diff
@@ -18,169 +18,173 @@ import sbt._
-    val uaParser         = "com.github.ua-parser"       %  "uap-java"                      % V.uaParser
-    val postgresDriver   = "org.postgresql"             %  "postgresql"                    % V.postgresDriver
-    val mysqlConnector   = "mysql"                      %  "mysql-connector-java"          % V.mysqlConnector
-    val jaywayJsonpath   = "com.jayway.jsonpath"        %  "json-path"                     % V.jaywayJsonpath
-    val yauaa            = "nl.basjes.parse.useragent"  %  "yauaa"                         % V.yauaa
-    val guava            = "com.google.guava"           %  "guava"                         % V.guava
-    val log4j            = "org.apache.logging.log4j"   % "log4j-core"                     % V.log4j
-    val log4jApi         = "org.apache.logging.log4j"   % "log4j-api"                      % V.log4j
-
-    val circeCore        = "io.circe"                   %% "circe-core"                    % V.circe
-    val circeGeneric     = "io.circe"                   %% "circe-generic"                 % V.circe
-    val circeExtras      = "io.circe"                   %% "circe-generic-extras"          % V.circe
-    val circeParser      = "io.circe"                   %% "circe-parser"                  % V.circe
-    val circeLiteral     = "io.circe"                   %% "circe-literal"                 % V.circe
-    val circeJava8       = "io.circe"                   %% "circe-java8"                   % V.circe
-    val circeJawn        = "io.circe"                   %% "circe-jawn"                    % V.circe
-    val circeConfig      = "io.circe"                   %% "circe-config"                  % V.circeConfig
-    val circeOptics      = "io.circe"                   %% "circe-optics"                  % V.circeOptics
-    val circeJackson     = "io.circe"                   %% "circe-jackson210"              % V.circeJackson
-    val scalaUri         = "io.lemonlabs"               %% "scala-uri"                     % V.scalaUri
-    val gatlingJsonpath  = "io.gatling"                 %% "jsonpath"                      % V.gatlingJsonpath
-    val scalaForex       = "com.snowplowanalytics"      %% "scala-forex"                   % V.scalaForex
-    val refererParser    = "com.snowplowanalytics"      %% "scala-referer-parser"          % V.refererParser
-    val maxmindIplookups = "com.snowplowanalytics"      %% "scala-maxmind-iplookups"       % V.maxmindIplookups
-    val scalaWeather     = "com.snowplowanalytics"      %% "scala-weather"                 % V.scalaWeather
-    val badRows          = "com.snowplowanalytics"      %% "snowplow-badrows"              % V.badRows
-    val igluClient       = "com.snowplowanalytics"      %% "iglu-scala-client"             % V.igluClient
-    val snowplowRawEvent = "com.snowplowanalytics"      %  "snowplow-thrift-raw-event"     % V.snowplowRawEvent
-    val collectorPayload = "com.snowplowanalytics"      %  "collector-payload-1"           % V.collectorPayload
```

Comment:
The whole file has been updated, it's impossible to know what is relevant to this test